### ISSUE TITLE
Documentation for Advanced/CLI/Prep is out of date

### ISSUE THREAD

**Issue by jkhartshorne** [pushkin-consortium/pushkin#61]
Needs to be updated to account for yalc.

**Comment by jessestorbeck** [comment]
@jkhartshorne Is this still relevant? I see stuff about yalc [here](https://languagelearninglab.gitbook.io/pushkin/developers/getting-started-on-development#yalc-for-pushkin-worker-pushkin-api-and-pushkin-client), but not in the docs for `prep`.

**Comment by jkhartshorne** [comment]
No, that's actually a separate issue. This one is probably still easiest for me to address.
